bayer2ppm

```diff
@@ -94,11 +97,49 @@ def bayer2ppm(config, infilespec):
         print(f"{_elapsed(t0)} - reading input file [{infilename}]: {width} x {height}, {config.bpp} bpp; range = [{np.min(raw)}, {np.max(raw)}]")
     image = isp(config, raw)
     t0 = time.time()
+
+    if config.outdir is not None:
+        outfilename = os.path.join(config.outdir, outfilename)
+
     imgio.imwrite(outfilename, image, config.maxval)
     if config.verbose:
         print(f"{_elapsed(t0)} - writing output file [{outfilename}]: {width} x {height}, {config.bpp} bpp")
         print(f"{_elapsed(tStart)} - TOTAL\n")
 
+    if config.process_yuv:
+        t1Start = t1 = time.time()
+        print(f"Converting {infilename} to YUV...")
+        yuv_image = rgb_to_yuv(config, image.copy())
+        print(f"{_elapsed(t1)} - converting RGB to YUV")
+
+        # Get Y-plane and scale it to 0-255 range
+        y_plane = yuv_image[:,:,0]
+        y_plane *= (255.0 / y_plane.max())
+        y_plane_data = y_plane.astype(np.uint8)
+        np.clip(y_plane_data, 0, 255)
```

Comment:
Clipping should be done before conversion to uint8. Doesn't make much sense to clip an 8-bit value to [0, 255]. :-)

Reply:
Oops. Lets assume it is a typo :)